modals/reservation/ReservationVM.ts

```diff
@@ -0,0 +1,13 @@
+export interface ReservationVM {
+  Id:number;
+  ContactName: string;
+  ContactPhoneNumber: string;
+  EmailId: string;
+  NumberOfGuests: number;
+  CreatedDate: string;
+  MealId: number;
+}
+ 
+export type Props = {
+  mealId: number
+};
```

Comment:
while this will work, i dont recommend exporting Props types to a separate file. it is more readable to see directly on top of the component .tsx file